generator_process/actions/depth_to_image.py

```diff
@@ -277,18 +281,19 @@ def __call__(
                     num_warmup_steps = len(timesteps) - num_inference_steps * self.scheduler.order
                     with self.progress_bar(total=num_inference_steps) as progress_bar:
                         for i, t in enumerate(timesteps):
+                            use_cfg = do_classifier_free_guidance and (i / len(timesteps)) < kwargs['cfg_end']
```

Comment:
I need to simplify this, CFG cannot be re-enabled partway through, so the `do_classifier_free_guidance` can just be modified when we surpass the end.